resources/options.js

```diff
@@ -68,31 +68,76 @@ function restore_options_availability() {
         // Default value
         loggingEnabled: true
     }, function(items) {
-        // The availability depends on the value of the logging checkbox state.
-        l('tabs').disabled = l('comments').disabled =
-            l('peer_comments').disabled = l('focus').disabled =
-                l('username').disabled = l('repo').disabled =
-                    l('file').disabled = !items.loggingEnabled;
+        var enabled = items.loggingEnabled;
+        if (enabled) {
+            show();
+        } else {
+            hide();
+        }
```

Comment:
Can you please document here that you can use different methods for show/hide (for the user testing)?